src/client/components/opt-in/opt-in.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div
+    v-if="!isAccepted"
+    role="dialog"
+    class="opt-in"
+    aria-labelledby="opt-in__title"
+    aria-describedby="opt-in__text"
+  >
+    <h2 class="a11y-sr-only" >{{title}}</h2>
+    <div id="opt-in__text" class="opt-in__text" v-html="body"></div>
+    <div class="opt-in__buttons">
+      <button
+        class="button button--primary"
+        @click="onAccept">
+        {{ $t('agree') }}
+      </button>
+      <button
+        class="button"
+        @click="onDecline"
+      >
+        {{ $t('decline') }}
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data () {
+    let isAccepted = 0
+    try {
+      isAccepted = localStorage.getItem(this.name)
+    } catch (e) {
+      isAccepted = 0
+    }
+    return { isAccepted }
+  },
+  props: {
+    name: String,
+    title: String,
+    body: String,
+  },
+  created () {
+    // Emit accept event if already accepted by user
+    if (this.isAccepted) {
+      this.$emit('accept')
+    }
+  },
+  methods: {
+    onAccept () {
+      if ('localStorage' in window) {
```

Comment:
Again, move the local storage bit to `lib/local-storage`? and why not use `true` and `false` instead of `1` and `0`. To me that makes it clear it's a boolean, not a number.